packages/paste-codemods/tools/.cache/mappings.json

```diff
@@ -20,6 +20,7 @@
   "Card": "@twilio-paste/core/card",
   "ChatBubble": "@twilio-paste/core/chat-log",
   "ChatMessage": "@twilio-paste/core/chat-log",
+  "MessageVariantContext": "@twilio-paste/core/chat-log",
```

Comment:
This is odd, I didn't modify anything in codemods, but I saw this when I went to commit changes. 
I think this component was recently added, but shouldn't this diff be in a different pull request? 🤔